bootloader/assets/data/grub.cfg

```diff
@@ -31,12 +32,17 @@ elif [ -n "$kernel_status" ]; then
     save_env kernel_status
 fi
 
+set cmdline_args="$snapd_static_cmdline_args $snapd_extra_cmdline_args"
```

Comment:
(nitpick^2) might be nice to keep things together similar to the recovery grub config, i.e. put the `set cmdline_args=` right after the `load_env` just like in recover grub. Mostly thinking that the same parts of the two files should look similar. But really a minor complaint :)

Reply:
Move that closed to load_env